internal/resources/policy/kind/common/constants.go

```diff
@@ -0,0 +1,11 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
```

Comment:
nit: copyright year 2022 -> 2023

Reply:
It happened at refactoring, extracted constants. I changed that.